lib/jekyll/utils/platforms.rb

```diff
@@ -13,18 +13,44 @@ module Platforms
         end
       end
 
+      #
+
+      def windows?
+        !!(RbConfig::CONFIG["host_os"] =~ %r!mswin|mingw|cygwin!i || \
```

Comment:
@XhmikosR @spudowiar we don't care if it's x64, as @spudowiar pointed out, that's why it's unanchored, we are only interested if we find a specific keyword.  There could be dozens of builds.